Documentation/marconi.rst

```diff
@@ -0,0 +1,276 @@
+  VERSION="openmpi4-1-x_ucx1-10-x"
+  export SEISSOL_INSTALL=$HOME/usr/local/${VERSION}
+
+  export PATH=$SEISSOL_INSTALL/bin:$PATH
+  export LIBRARY_PATH=$SEISSOL_INSTALL/lib:$LIBRARY_PATH
+  export LD_LIBRARY_PATH=$SEISSOL_INSTALL/lib:$LD_LIBRARY_PATH
+  export PKG_CONFIG_PATH=$SEISSOL_INSTALL/lib/pkgconfig:$PKG_CONFIG_PATH
+  export CMAKE_PREFIX_PATH=$SEISSOL_INSTALL
+  export CPATH=$SEISSOL_INSTALL/include:$CPATH
+  export CPLUS_INCLUDE_PATH=$SEISSOL_INSTALL/include:$CPLUS_INCLUDE_PATH
+  export C_INCLUDE_PATH=$SEISSOL_INSTALL/include:$C_INCLUDE_PATH
+
+2. Load basic modules, source launch file and create an install directory:
+
+::
+
+  $ source ./launch.sh
+  $ mkdir -p $SEISSOL_INSTALL
+
+  $ module load gnu/8.4.0
+  $ module load cmake/3.20.0
+  $ module load python/3.8.2
+
+3. Create any directory where you are going to configure and build libraries and packages. For example, 
+
+::
+
+  mkdir -p $HOME/Download
+  cd $HOME/Download
+
+4. Install UCX:
+
+::
+
+  $ git clone https://github.com/openucx/ucx.git
+  $ cd ucx
+  $ git checkout v1.10.x
+  $ ./autogen.sh
+  $ mkdir build && cd build
+
+  $ CC=$(which gcc) CXX=$(which g++) FC=$(which gfortran) \
+  ../contrib/configure-opt \
+  --prefix=$SEISSOL_INSTALL \
+  --build=powerpc64le-redhat-linux-gnu \
+  --host=powerpc64le-redhat-linux-gnu \
+  --disable-logging --disable-debug \
+  --disable-assertions --disable-params-check \
+  --disable-params-check --enable-optimizations \
+  --disable-assertions --disable-logging --with-pic \
+  --without-java \
+  --enable-mt \
+  --with-cuda=/cineca/prod/opt/compilers/cuda/11.0/none \
+  --with-gdrcopy \
+  --with-knem=/opt/knem-1.1.3.90mlnx1 \
+  --without-xpmem
+
+  $ make -j
+  $ make install
+  $ cd ../..
+
+5. Install OpenMPI:
+
+::
+
+  $ git clone --recursive -b v4.1.x https://github.com/open-mpi/ompi.git
+  $ cd ompi
+  $ ./autogen.pl
+  $ mkdir build && cd build
+
+  $ CC=$(which gcc) CXX=$(which g++) FC=$(which gfortran) \
+  CFLAGS="-I/opt/pmix/3.1.5/include" CPPFLAGS="-I/opt/pmix/3.1.5/include" \
+  ../configure \
+  --prefix=$SEISSOL_INSTALL \
+  --with-memory-manager=none \
+  --enable-static=yes \
+  --enable-shared \
+  --with-slurm \
+  --with-pmix=/opt/pmix/3.1.5 \
+  --with-ucx=$SEISSOL_INSTALL \
+  --with-libevent=/usr \
+  --with-hwloc=/usr \
+  --with-verbs \
+  --enable-mpirun-prefix-by-default \
+  --with-platform=/cineca/prod/build/compilers/openmpi/4.0.3/gnu--8.4.0/BA_WORK/openmpi-4.0.3/contrib/platform/mellanox/optimized \
+  --with-hcoll=/opt/mellanox/hcoll \
+  --with-cuda=/cineca/prod/opt/compilers/cuda/11.0/none \
+  --with-knem=/opt/knem-1.1.3.90mlnx1 \
+  --without-xpmem
+
+  $ make -j
+  $ make install
+  $ cd ../..
+
+6. Install HDF5:
+
+::
+
+  $ wget https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.10/hdf5-1.10.5/src/hdf5-1.10.5.tar.gz
+  $ tar -xvf ./hdf5-1.10.5.tar.gz
+  $ cd hdf5-1.10.5
+  $ ./autogen.sh
+  $ mkdir build && cd build
+
+  $ CPPFLAGS="-fPIC ${CPPFLAGS}" CC=mpicc CXX=mpicxx FC=mpif90 \
```

Comment:
Shouldn't that be CFLAGS instead of CPPFLAGS?

Reply:
It seems to me that I copied and pasted it from `Installing HDF5`. I guess yes. In my case it didn't break my installation on M100.